modules/rostests/apitests/rtl/CMakeLists.txt

```diff
@@ -38,9 +38,6 @@ add_dependencies(rtl_test_lib asm)
 
 if(NOT MSVC)
     set_source_files_properties(RtlGetFullPathName_UstrEx.c PROPERTIES COMPILE_FLAGS "-Wno-format")
-
-    # Avoid "universal character names are only valid in C++ and C99" error.
-    set_property(TARGET rtl_test_lib PROPERTY C_STANDARD 99)
```

Comment:
There is another occurrence at
https://github.com/reactos/reactos/blob/a549a9d8e403180f42e84e9c3e7979d60e62c268/modules/rostests/apitests/ntdll/CMakeLists.txt#L142-L143